Model/rebelShip.php

```diff
@@ -13,7 +16,13 @@ public function getFavoriteJedi()
 
     public function getType()
     {
-        return AbstractShip::REBEL;
+        // return AbstractShip::REBEL;
```

Comment:
See note in AbstractShip about changing this back

Reply:
Done